src/Momento.Sdk/Incubating/Internal/ScsDataClient.cs

```diff
@@ -388,15 +386,17 @@ public async Task SendListPushBackAsync(string cacheName, string listName, ByteS
             RefreshTtl = refreshTtl,
             TtlMilliseconds = TtlSecondsToMilliseconds(ttlSeconds)
         };
+        _ListPushBackResponse response;
 
         try
         {
-            await this.grpcManager.Client.ListPushBackAsync(request, MetadataWithCache(cacheName), deadline: CalculateDeadline());
+            response = await this.grpcManager.Client.ListPushBackAsync(request, MetadataWithCache(cacheName), deadline: CalculateDeadline());
         }
         catch (Exception e)
         {
             throw CacheExceptionMapper.Convert(e);
         }
```

Comment:
maybe null check on response before returning? idk if desired or not just calling out incase.

Reply:
This will throw an exception before returning null

Reply:
ok sounds good! bit hard to see that from the outside on GH but sounds good.